### PR TITLE
Added game borrowing

### LINKED ISSUE #18: As a gamer, I would like to know if games already are borrowed or not, so I know which games that are availble for me.

### PR DESCRIPTION
**Pull request by x183**
Added indicators on game borrowing status, fixes #18

### CODE REVIEW

**Comment by x183** [comment]
Please squash this when merging

**Review by GAsplund — APPROVED** [review]
LGTM!

File: backend/src/routers/borrowRouter.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+const borrowSchema = z.object({
+	gameId: z.string().min(1),
+	user: z.string().min(1),
+	borrowStart: z.string().datetime(),
+	borrowEnd: z.string().datetime()
+});
+
+/**
+ * @api {post} /api/v1/games/borrow Borrow a game
```
Comment (Oscariremma) [inline backend/src/routers/borrowRouter.ts:16]: Correct endpoint in apidoc
```suggestion
 * @api {post} /api/v1/borrow Borrow a game
```

File: backend/src/routers/borrowRouter.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	res.status(200).json({ message: 'Game successfully borrowed' });
+});
+
+const returnSchema = z.object({
+	gameId: z.string().min(1),
+	user: z.string().min(1)
+});
+
+/**
+ * @api {post} /api/v1/games/return Returns a borrowed game
```
Comment (Oscariremma) [inline backend/src/routers/borrowRouter.ts:53]: Correct endpoint in apidoc
```suggestion
 * @api {post} /api/v1/borrow/return Returns a borrowed game
```

**Review by Oscariremma — CHANGES_REQUESTED** [review]

**Review by Oscariremma — APPROVED** [review]
Looks good!

**Comment by Supergamer1337** [comment]
I know this got merged, but why can I borrow games that don't exist?

**Comment by Supergamer1337** [comment]
I can also borrow something that is already borrowed?

**Comment by x183** [comment]
> I can also borrow something that is already borrowed?

Yes, and return something twice 😃

**Pull request merged**